src/Sylius/Component/Grid/Renderer/GridRendererInterface.php

```diff
@@ -48,6 +48,15 @@ public function renderAction(GridViewInterface $gridView, Action $action, $data
 
     /**
      * @param GridViewInterface $gridView
+     * @param Action $bulkAction
+     * @param mixed|null $data
+     *
+     * @return mixed
+     */
+    public function renderBulkAction(GridViewInterface $gridView, Action $bulkAction, $data = null);
```

Comment:
It's a BC break for sure, we need to revert these changes and think of a better, backwards compatibility solution.